home_first_aid_kit/lib/storage.ex

```diff
@@ -0,0 +1,44 @@
+defmodule HomeFirstAidKitStorage do
+  @moduledoc """
+  Documentation for `HomeFirstAidKitStorage`.
+  """
+
+  @doc """
+  Get storage path
+  ## Examples
+      iex> HomeFirstAidKitStorage.get_storage_path()
+      ".data/home_first_aid_kit.json"
+
+  """
+  def get_storage_path() do
+    Application.get_env(:home_first_aid_kit, :data_dir) <> "/home_first_aid_kit.json"
+  end
+
+  @doc """
+  Read from storage file and parse json
+  """
+  def read_json(filename) do
+    File.read(filename) |> case do
```

Comment:
```suggestion
    filename
    |> File.read()
    |> case do
```

Включайте в редакторе (или запускайте) `mix format`.